src/background.js

```diff
@@ -239,13 +240,25 @@ const rollout = {
 
     switch (typeof defaultValue) {
     case "boolean":
-      log("boolean", name, await browser.experiments.preferences.getBoolPref(name, defaultValue));
+      log({
+        "type": "boolean",
+        "name": name,
```

Comment:
nit: this can just be `name` ("shorthand property names" in https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Object_initializer)